### ISSUE TITLE
Proposal: support for additional clauses

### ISSUE THREAD

**Issue by mihar-22** [terkelg/sqliterally#8]
Support for the following clauses would make this library closer to feature complete in my opinion:

**insert**

api: `query.insert('tableName')`

output: `INSERT INTO tableName`

**column**

api: `query.column('columnOne').column('columnTwo')` 

output: `(columnOne, columnTwo)`

**value**

api: `query.value('valueOne').value('valueTwo')` 

output: `VALUES (valueOne, valueTwo)`

**insert + column + value**

api: `query.insert('tableName').column('columnOne').column('columnTwo').value('valueOne').value('valueTwo')`

output: `INSERT INTO tableName (columnOne, columnTwo) VALUES (valueOne, valueTwo)`

**delete**

api: `query.delete('tableName')`

output: `DELETE FROM tableName`

**Comment by terkelg** [comment]
I agree this could be nice. I haven't worked with Postgres in a while or had the need for this feature yet. I'm open to any PR that would implement this, but I won't have time to implement this myself
